packages/components-js/tests/vrt/playwright/specs/common/components.vrt.ts

```diff
@@ -128,6 +128,8 @@ components.forEach((component) => {
 
     // 200% font scaling
     test(`should have no visual regression for viewport ${baseViewportWidth} in scale mode`, async ({ page }) => {
+      test.skip(component === 'select-wrapper', 'select-wrapper is flaky');
```

Comment:
Do we want to skip this test for now?

Reply:
I'd say yes, better than retrying 10 times 😢